Source/Core/DolphinQt/Config/Mapping/HotkeyControllerProfile.cpp

```diff
@@ -16,10 +16,27 @@ HotkeyControllerProfile::HotkeyControllerProfile(MappingWindow* window) : Mappin
 
 void HotkeyControllerProfile::CreateMainLayout()
 {
-  m_main_layout = new QHBoxLayout();
-
-  m_main_layout->addWidget(CreateGroupBox(
-      tr("Controller Profile"), HotkeyManagerEmu::GetHotkeyGroup(HKGP_CONTROLLER_PROFILE)));
+  QGridLayout* m_main_layout = new QGridLayout;
```

Comment:
Please remove m_ from this variable name.
You can also declare the type as const auto.

Reply:
Done.